packages/server/src/automations/actions.ts

```diff
@@ -107,10 +107,12 @@ export async function getActionDefinitions() {
 }
 
 /* istanbul ignore next */
-export async function getAction(stepId: string) {
+export async function getAction(stepId: AutomationActionStepId) {
   if (ACTION_IMPLS[stepId] != null) {
     return ACTION_IMPLS[stepId]
   }
+  ACTION_IMPLS["CREATE_ROW"]
```

Comment:
Not sure this statement does anything!

Reply:
I just noticed that this morning, I was testing the typing and forgot to remove it!